src/Components/Form/Form.js

```diff
@@ -44,17 +50,21 @@ export class Form extends Component {
 
   determinePrompt = (index, data) => {
     return (
-        <article className='question-with-choices'>
-          <h2 className='single-question'>{data[index].question}</h2>
-          <div>
+        <article className='form-container'>
+          <h2 className='question'>{data[index].question}</h2>
+          <div className='possible-answers'>
             {data[index].choices.map((choice, i) => {
-              return <h2
-                key={i}
-                id={data[index].answerType}
-                onClick={this.updateCurrentAnswers} 
-                value={choice} 
-                className='choice'>{choice}
-                </h2>
+              return (
+                <div className='choice'>
+                  <h3
+                    key={i}
+                    id={data[index].answerType}
+                    onClick={this.updateCurrentAnswers} 
+                    value={choice} 
+                    className='option'>{choice}
+                  </h3>
+                </div>
+                )
```

Comment:
We will also probably run into conflicts here regarding the anonymous function on my end, we will have to talk about what makes more sense.